packages/@uppy/react-native/package.json

```diff
@@ -22,15 +22,15 @@
   },
   "dependencies": {
     "@uppy/instagram": "workspace:^",
-    "@uppy/url": "workspace:^"
-  },
-  "peerDependencies": {
-    "expo": ">=33.0.0",
+    "@uppy/url": "workspace:^",
     "expo-document-picker": ">=6.0.0",
     "expo-image-picker": ">=6.0.0",
     "expo-permissions": ">=6.0.0",
-    "react": "^16.0.0 || ^17.0.0 || ^18.0.0",
-    "react-native": "*",
     "react-native-super-grid": "*"
```

Comment:
these needs clearly to be deps and not peerdeps, you even receive an error if you try to import uppy/react-native without directly installing also these expo libs

Reply:
I think the idea behind having those as peer deps is that it would let users use whichever version suit them. We probably need to have them in our dev deps though.

Reply:
> Dev deps are for code not used during runtime, so this ain't the case.

I don't think that's true, it's for deps that are needed for development and testing. And we probably need react-native and react for testing if the lib works.

Reply:
@aduh95 Development and testing aren't things done on runtime... Btw I'm talking about the expo libs, not react and react-native which are clearly peerdeps

Reply:
By the way, the matter is indeed very confusing and not straightforward as I thought. I've inspected moments ago a lot of libraries that use Axios, for example, and they put it under dependencies but reading this 👉 https://nodejs.org/es/blog/npm/peer-dependencies/ seems clear they should go back in the peerDependencies section.

I'll do it asap, thanks 🙏

Reply:
> Development and testing aren't things done at runtime

What I meant was the fact that the runtime may or may not use a dep is irrelevant, devDeps should contain all the things needs for development and testing. Anyway it's a small nuance, let's not fight over this.



> Btw I'm talking about the expo libs, not react and react-native which are clearly peerdeps

Oh ok I didn't get that. In that case, I don't have an opinion, please disregard my previous comments, you probably know more than I do on the subject.

Reply:
I wonder if @Murderlon also thought the same! Sadly GitHub's diffing is a bit confusing, but I've actually moved expo libs from peerdeps to deps, **not** react, react-native and expo

Reply:
after a little convo on twitter, those expo libs definitely need to go under dependencies for the last point of this [post](https://stackoverflow.com/questions/26737819/why-use-peer-dependencies-in-npm-for-plugins): 
"You expect/need the user to work with that other library as well", the answer is no.

We definitely don't think nor know if the users of this lib would use those expo libs

Reply:
I also checked the expo documentation: https://docs.expo.dev/versions/latest/sdk/imagepicker/
They recommend putting it in deps, not peer deps, so it looks like there's a consensus on the issue.

Reply:
They recommend setting up the project with `npx create-expo-app my-app` and to add deps with `npx expo install expo-image-picker`, which add it as deps, not peer deps.

Reply:
oh yes, but that is for apps and not libs sadly!